src/mdg/config.yml

```diff
@@ -0,0 +1,13 @@
+iso2_code: mg
```

Comment:
just make sure to merge this file with `feature/plague_explore`'s version to enable merging

Reply:
Decided not to use so will actually remove from this branch.